src/librustc_resolve/error_codes.rs

```diff
@@ -1013,33 +1013,40 @@ fn h1() -> i32 {
 "##,
 
 E0424: r##"
-The `self` keyword was used in a static method.
+The `self` keyword was used inside of an associated function without a "`self`
+receiver" parameter. The `self` keyword can only be used inside methods, which
```

Comment:
Please put the explananation **after** the failing example.

Reply:
Done.

Reply:
Done.